### DISCUSSION TITLE
2x zigbee2mqtt in one HA installation

### DISCUSSION THREAD

**Discussion by satelitman** [Koenkk/zigbee2mqtt#17083] [General]
HELLO FRIENDS. I am turning to you with a question. It is possible to run 2x zigbee2mqtt in one HA installation. There is a video on YouTube where a guy claims that it can be done. I followed his recommendations. It almost worked, but after a reboot (HA hardware reboot) it stopped working. In that manual, it is said that during configuration it is necessary to change the port and pan ID. I'm not sure here, as he sets it up via (configuration.yaml) in his communication, but in the current version, I use (Current version: 1.30.2-1) I was able to change the port during the second installation during the configuration, but Mr. ID can change only after installing in advanced settings. Why do I need it??? I have a problem with the IKEA tradfri, which automatically connects to the plug (lumi.plug.mmeu01) and can turn it off!!! MY HEATING BOILER SHUTS OFF !!! - , while I was using IKEA to control other things. Unbinding stopped working. And so I bought one more key based on the recommendation from the video that I will create a kind of network where I will only put IKEA so that it does not connect to any of mine (lumi.plug.mmeu01).

QUESTION:
01. Is it at all possible to run 2x zigbee2mqtt ???

02. It can't be a problem that I use the first key - connected to USB with a chip - CC2652P and the second installation I connect via ethernet with the SMLIGHT SLZB-06 key BUT !!! ALSO WITH THE CC2652P CHIP - IS IT NOT A PROBLEM THAT TWO SAME CHIPS ARE CONNECTED ???

03. AND IF IT IS POSSIBLE, IT IS ENOUGH TO SET A DIFFERENT PORT DURING THE INSTALLATION than it is during the first installation (default 8485) and in the second I will put the port, for example, 8486 ???

04. how to set a different PAN ID in the second installation of zigbee2mqtt ???
Thank you in advance for any answer and help.

Excuse my english.

**Comment by sergge1** [comment]
@satelitman 
1. Yes
2. Not a problem
3. No. should be different Port for connection, PAN_ID and EXTENDED PAN_ID (also it is recommended to use different Zigbee channels)
4. https://www.zigbee2mqtt.io/guide/configuration/zigbee-network.html#network-config

Have a look here as well:
https://www.reddit.com/r/homeassistant/comments/z28f9b/my_experience_with_multiple_zigbee_poe/
